extensions-core/s3-extensions/pom.xml

```diff
@@ -46,16 +46,16 @@
             <scope>provided</scope>
         </dependency>
         <dependency>
-            <groupId>io.druid</groupId>
-            <artifactId>java-util</artifactId>
-            <version>${project.parent.version}</version>
-            <scope>provided</scope>
-        </dependency>
-        <dependency>
-            <groupId>net.java.dev.jets3t</groupId>
-            <artifactId>jets3t</artifactId>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-java-sdk-s3</artifactId>
```

Comment:
is this still needed since aws-common pulls in the bundle?

Reply:
Yes, maybe. I didn't test yet.